### ISSUE TITLE
Update ...success-color--200 to call ...green--600

### ISSUE THREAD

**Issue by mceledonia** [patternfly/patternfly#3314]
Currently `success-200` calls `green-700` which reads as black rather than dark green. `Green-600` reads more as a dark green and still passes accessibility requirements in our use cases. 

![image](https://user-images.githubusercontent.com/30935938/88059799-5bac2380-cb33-11ea-9ee2-1523c6ff1f28.png)
